index.html

```diff
@@ -0,0 +1,382 @@
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="contact-form">
+        <section class="contact-form-top-section">
+          <h2>
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+          </h2>
+
+          <form id="myForm" class="form">
+            <input name="name" id="fullName" type="text" placeholder="Full name" maxlength="30">
+            <input name="Email" id="email" type="text" placeholder="Email address">
+            <textarea name="text-area" id="comment" placeholder="Enter your text here" cols="30" rows="6" maxlength="500"></textarea>
+            <small></small>
+            <button class="get-it-touch-btn">Get It Touch</button>
+        </form>
```

Comment:
- [ ] Kindly change the button to say 'Get in touch' instead of 'Get it touch'.
- [ ] Please add the button interactions (hover, active, ...etc) to the form button.

Reply:
Kindly implement changes, from the previous review. The information on the button is still `Get it touch`, change it to `Get in touch ` ; when I hover over the button, there is no `hover effect `.